src/NServiceBus.Gateway.AcceptanceTests/When_listening_on_multiple_uris.cs

```diff
@@ -0,0 +1,136 @@
+﻿namespace NServiceBus.Gateway.AcceptanceTests
+{
+    using System;
+    using System.IO;
+    using System.Net;
+    using System.Security.Cryptography;
+    using System.Text;
+    using System.Threading.Tasks;
+    using System.Web;
+    using AcceptanceTesting;
+    using NUnit.Framework;
+
+    public class When_listening_on_multiple_uris : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_receive_on_all_uris()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Headquarters>(b => b.When(bus =>
+                {
+                    var hostname = Dns.GetHostName();
+
+                    SendMessage(DefaultReceiveURI);
+                    SendMessage($"http://{hostname}:25898/Headquarters/");
+                    return Task.FromResult(0);
+                }))
+                .Done(c => c.GotMessageOnDefaultChannel && c.GotMessageOnNonDefaultChannel)
+                .Run();
+
+            Assert.IsTrue(context.GotMessageOnDefaultChannel);
+            Assert.IsTrue(context.GotMessageOnNonDefaultChannel);
+        }
+
+        void SendMessage(string url)
+        {
+            var webRequest = CreateWebRequest(url);
+
+            while (true)
+            {
+                try
+                {
+                    using (var myWebResponse = (HttpWebResponse)webRequest.GetResponse())
+                    {
+                        if (myWebResponse.StatusCode == HttpStatusCode.OK)
+                        {
+                            break;
+                        }
+                    }
+                }
+                catch (WebException)
+                {
+                }
+            }
+        }
+
+        static HttpWebRequest CreateWebRequest(string uri)
+        {
+#pragma warning disable DE0003 // API is deprecated
```

Comment:
is this API really deprecated? removing the comment locally doesn't seem to cause any issues.